### DISCUSSION TITLE
Do we have a target resolver thing for AWS in cloudprober?

### DISCUSSION THREAD

**Discussion by tianqiliu-RBLX** [google/cloudprober#675] [Q&A]
Cc @manugarg ^

Thanks!

**Comment by manugarg** [comment]
@tianqiliu-RBLX We don't, at the moment.

**Comment by manugarg** [comment]
@tianqiliu-RBLX Please note this repository's active development has now moved to github.com/cloudprober/cloudprober. We'll be abandoning/archiving this location.
